.gitignore

```
@@ -25,3 +25,4 @@
 # Helm
 kubernetes/linera-validator/working/*
 node_modules
+**dist/
```

```suggestion
**/dist/
```

I think this one will exclude anything ending in the string `dist`, and we probably only want to exclude directories called `dist`.